src/main/resources/application.yml

```
@@ -22,4 +22,7 @@ spring:
 app:
   batch:
     aimodel-index:
-      enabled: true  # AIModel 인덱싱 스케줄러 활성화 (증분 처리)
\ No newline at end of file
+      enabled: true  # AIModel 인덱싱 스케줄러 활성화 (증분 처리)
+  auth:
+    oauth2:
+      enabled: false  # OAuth2 기본 비활성화
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)